### DISCUSSION TITLE
Copilot extension doesn't work in my second PC

### DISCUSSION THREAD

**Discussion by MatallahMouncif** [community/community#37106] [Copilot Conversations]
I saw all the problems but I see that no one had the same issue as me, I have two computers I work with the same github account, the problem is that in my second PC it connect but it's always COPILOT is encountering temporarily issues., and it is the same problem in all the editors of my pc (VSC, intellij, Android Studio)

**Comment by ossanna16** [comment]
Hi there @MatallahMouncif and welcome to our community! Thank you for asking a great question 🙂
